br/pkg/lightning/config/config.go

```diff
@@ -143,6 +144,8 @@ type Config struct {
 	Cron         Cron                `toml:"cron" json:"cron"`
 	Routes       []*router.TableRule `toml:"routes" json:"routes"`
 	Security     Security            `toml:"security" json:"security"`
+	CheckOnlyCfg *CheckOnlyConfig    `toml:"-" json:"-"`
```

Comment:
Let's remove the `cfg` and `config` suffix since we are under the config package.

Reply:
Why set tags to `-`? I think we can also configure check-only config through a config file.

Reply:
it's command line only